Blish HUD/Controls/Panel.cs

```diff
@@ -197,7 +197,7 @@ public class Panel : Container, IAccordion {
         }
 
         private void UpdateContentRegionBounds(object sender, EventArgs e) {
-            //UpdateScrollbar();
+            UpdateScrollbar();
```

Comment:
did you test that this doesn't have any ugly side effects? do you remember why it was commented out in the first place?

Reply:
Yes, per the git blame, it looks like I commented it out 9 months ago.  It covers the case where a panel has `CanScroll` enabled but does not have enough content to necessitate it scrolling.  If a new control is added that is either larger than or extends passed the content region necessitating a scroll bar, a scroll bar is given the chance to be instanced.